lib/Sabre/VObject/Property.php

```diff
@@ -195,6 +195,15 @@ public function serialize() {
             '\n',
             '',
         );
+
+        // avoid double-escaping of \, and \; from Compound properties
+        if (method_exists($this, 'setParts')) {
```

Comment:
The reason why I check by method is that the replacements below revert the escape characters that get added in `setParts()`. IMO the check is just right.